cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -2252,6 +2272,11 @@ request.ui_dispatch_view = search
 search = `cs_o365` sourcetype="o365:management:activity" _index_earliest=-61m@m _index_latest=-1m@m Workload=AzureActiveDirectory Operation=UserLoggedIn  `cs_public_ips(ClientIP)`  NOT LogonError=* \
 | stats count, values(user_type) as user_type, values(authentication_method) as authentication_method, max(_time) as Last_Success_Login by user, ClientIP, app \
 | iplocation ClientIP \
+| eval main_event="1" \
```

Comment:
I can do that but in the future if we decide to use some other product/alert specific field in the stats/by then it would be hard to do. that's why i have not combined it.

Reply:
Right at that time we can use direct way. Macro is to make it easier not necessary to use always.